src/client/renderingengine.cpp

```diff
@@ -410,6 +410,26 @@ void RenderingEngine::_draw_load_screen(const std::wstring &text,
 	guitext->remove();
 }
 
+/*
+	Draws the menu scene including (optional) cloud background.
+*/
+void RenderingEngine::_draw_menu_scene(gui::IGUIEnvironment *guienv,
```

Comment:
Please call this function in `RenderingEngine::_draw_load_screen` to prevent code duplication.